Assets/Scripts/Audio/FmodChordInterpreter.cs

```diff
@@ -97,6 +97,19 @@ public bool IsFmodMarkerChordInformation(string marker)
             return marker[0] == fmodFlagChar;
         }
 
+        // TODO: Relocate this to some sort of string builder util class.
```

Comment:
Should be changed to StringBuilder.Replace() like we talked about.

Reply:
Other than that, this like really nice. Good Job!

Reply:
Removed the RemoveSpacesFromStringBuilder function, now we just use StringBuilder.Replace().